### DISCUSSION TITLE
Ошибка ModuleNotFoundError: No module named 'telegram'

### DISCUSSION THREAD

**Discussion by Sanset-Sin** [python-telegram-bot/python-telegram-bot#3784] [Q&A] [answered]
Привет! 

Встал на пути создания бота для телеграмм, но вот незадача, при запуске кода выходит ошибка: 

Traceback (most recent call last):
  File "D:\main.py", line 1, in <module>
    from telegram import Update
ModuleNotFoundError: No module named 'telegram'

Process finished with exit code 1

Ошибка состоит в том, что не отслеживается основной модуль для работы с телеграмм ботом, но ОН УСТАНОВЛЕН и ОБНОВЛЕН до актуальной версии. В чем проблема я не могу понять. Пробовал переписать код, но выходит та же ошибка. Помогите решить проблему ,

Заранее благодарен.

**Comment by Poolitzer** [comment, marked as answer]
Hey, this is the internet, so we speak english unless otherwise allowed which it is not here.


https://github.com/python-telegram-bot/python-telegram-bot/wiki/Frequently-Asked-Questions#why-am-i-getting-importerror-cannot-import-name-xy-from-telegram
